src/index.html

```diff
@@ -18,5 +18,60 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="image">
+        <img
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </div>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="index.html"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a href="index.html">Samsung</a>
+          </li>
+
+          <li>
+            <a href="index.html">Smartphones</a>
+          </li>
+
+          <li>
+            <a
+              href="index.html"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a href="index.html">Gadgets</a>
+          </li>
+
+          <li>
+            <a href="index.html">Tablets</a>
```

Comment:
The `href` attribute for this link is set to `index.html`. Ensure it points to the correct page for 'Tablets'.